packages/lesmis/src/views/DashboardReportTabView.js

```diff
@@ -120,12 +118,20 @@ const useStickyBar = () => {
   };
 };
 
-export const DashboardReportTabView = ({ entityCode, TabSelector }) => {
-  const [selectedYear, setSelectedYear] = useUrlSearchParam('year', DEFAULT_DATA_YEAR);
+export const DashboardReportTabView = ({
+  entityCode,
+  TabBarLeftSection,
+  year,
+  filterSubDashboards,
+}) => {
   const [selectedDashboard, setSelectedDashboard] = useUrlSearchParam('subDashboard');
   const { data, isLoading, isError, error } = useDashboardData(entityCode);
   const { scrollToTop, topRef, isScrolledPastTop, onLoadTabBar } = useStickyBar();
-  const activeDashboard = useDefaultDashboardTab(selectedDashboard, data);
+  const subDashboards = useMemo(() => data && data.filter(filterSubDashboards), [
```

Comment:
```suggestion
const subDashboards = useMemo(() => data?.filter(filterSubDashboards), [
```
Minor but I think this should be fine to do...

Reply:
Oh good call! Love this syntax but still getting in the habit